### DISCUSSION TITLE
Décimal Places

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#7999] [Feature Requests]
Good afternoon, it can be very easy but I don't see how I can paste the column to always show 2 decimal places... I don't want hide the zeros

some help?

the field is even set to 10.5

![image](https://user-images.githubusercontent.com/81326296/133128684-999d5136-293a-49b6-884f-d558862801b5.png)

**Comment by rijkvanzanten** [comment]
That's not currently possible

**Reply by hugopvl** [reply]
But will it be possible in the future?

For those reading the data it creates some confusion as it is now.

**Reply by benhaynes** [reply]
The display must be rounding the data output, it makes perfect sense to add a precision option to a numerical display type. This feature request should cover it, but I'd _almost_ consider it a bug since it's not displaying the actual value.

What display is this field using? Can you try raw?

**Reply by hugopvl** [reply]
Yes this solution would be perfect, but as you comment we can see this with a bug, because it doesn't really show the real value, the image I present above is already in raw.

Complete example below

![image](https://user-images.githubusercontent.com/81326296/133250063-69bb3b9a-4663-4b38-ad0f-b04034dd2da5.png)

![image](https://user-images.githubusercontent.com/81326296/133249499-958b4520-e627-402f-9dad-05b533af8fb8.png)

![image](https://user-images.githubusercontent.com/81326296/133249242-8f5e9ee7-839e-48c7-8f80-7739cdde54d0.png)

![image](https://user-images.githubusercontent.com/81326296/133249801-e265e3a7-2e55-42b6-ac08-89d31b68e2c9.png)

**Comment by benhaynes** [comment]
It seems like the `raw` Display gives you the exact value, but should automatically respect the precision and scale of the datatype. We could also add more fine-grain controls to a dedicated `numeric` Display. Does that sound about right?

**Reply by hugopvl** [reply]
Yes, it shows the exact value, but it doesn't respect the precision scale, as you confirmed. to be perfect, either I should respect the precision scale or a numerical control for the display, this last one seems to me the best option, it gives you a little more flexibility.

**Reply by rijkvanzanten** [reply]
> It seems like the raw Display gives you the exact value, but should automatically respect the precision and scale of the datatype.

The raw value displays whatever the API returns "raw". In this case, JS doesn't have trailing zeros in float values, so `1.10000` will be returned as `1.1` in JSON, which causes the app to display it as such 🤔 I think we should add this precision and scale support to the `formatted-value` display to ensure raw remains raw.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
